Disable USB autosuspend and runtime PM modules

This functionality is now implemented in udev rules

BUG=chromium:686168, chromium:214886

--- laptop-mode-tools-1.59/etc/laptop-mode/conf.d/runtime-pm.conf
+++ laptop-mode-tools-1.59/etc/laptop-mode/conf.d/runtime-pm.conf
@@ -20,7 +20,7 @@
 ###############################################################################
 
 # Control Runtime Power Management ?
-CONTROL_RUNTIME_PM="auto"
+CONTROL_RUNTIME_PM="0"
 
 # Enable debug mode for this module
 # Set to 1 if you want to debug this module
--- laptop-mode-tools-1.59/etc/laptop-mode/conf.d/usb-autosuspend.conf
+++ laptop-mode-tools-1.59/etc/laptop-mode/conf.d/usb-autosuspend.conf
@@ -24,7 +24,7 @@
 
 # Enable USB autosuspend feature?
 # Set to 0 to disable
-CONTROL_USB_AUTOSUSPEND=1
+CONTROL_USB_AUTOSUSPEND=0
 
 # Set this to use opt-in/whitelist instead of opt-out/blacklist for deciding
 # which USB devices should be autosuspended.
--- laptop-mode-tools-1.59/etc/rules/99-laptop-mode.rules
+++ laptop-mode-tools-1.59/etc/rules/99-laptop-mode.rules
@@ -1,4 +1,3 @@
 ACTION=="change", SUBSYSTEM=="power_supply", RUN+="/lib/udev/lmt-udev auto"
 ACTION=="add|remove", SUBSYSTEM=="machinecheck", RUN+="/lib/udev/lmt-udev auto"
-ACTION=="add", SUBSYSTEM=="usb", RUN+="/lib/udev/lmt-udev force modules=usb-autosuspend devices=%k"
 ACTION=="add", SUBSYSTEM=="net", RUN+="/lib/udev/lmt-udev force modules=wifi-nl80211-powermgmt"
